allennlp/sanity_checks/task_checklists/textual_entailment_suite.py

```diff
@@ -0,0 +1,352 @@
+            remove_duplicates=True,
+        )
+
+        test = MFT(
+            **template,
+            labels=self._contradicts,
+            name='"A is COMP than B" contradicts "B is COMP than A"',
+            capability="Logic",
+            description='Eg. "A is better than B" contradicts "B is better than A"',
+        )
+
+        self.add_test(test)
+
+        if data:
+            template = Perturb.perturb(
+                data, lambda x: (x[0], x[0]), nsamples=num_test_cases, keep_original=False
+            )
+            template += Perturb.perturb(
+                data, lambda x: (x[1], x[1]), nsamples=num_test_cases, keep_original=False
+            )
+
+            test = MFT(
+                **template,
+                labels=self._entails,
+                name="A entails A (premise == hypothesis)",
+                capability="Logic",
+                description="If premise and hypothesis are the same, then premise entails the hypothesis",
+            )
+
+            self.add_test(test)
+
+    def _default_negation_tests(self, data: Optional[Iterable[Tuple]], num_test_cases=100):
+
+        template = self.editor.template(
+            (
+                "{first_name1} is {compare} than {first_name2}",
+                "{first_name1} is not {compare} than {first_name2}",
+            ),
+            nsamples=num_test_cases,
+            remove_duplicates=True,
+        )
+
+        test = MFT(
+            **template,
+            labels=self._contradicts,
+            name='"A is COMP than B" contradicts "A is not COMP than B"',
+            capability="Negation",
+            description="Eg. A is better than B contradicts A is not better than C",
+        )
+
+        self.add_test(test)
+
+    def _default_ner_tests(self, data: Optional[Iterable[Tuple]], num_test_cases=100):
+        template = self.editor.template(
+            (
+                "{first_name1} is {compare} than {first_name2}",
+                "{first_name1} is {compare} than {first_name3}",
+            ),
+            nsamples=num_test_cases,
+            remove_duplicates=True,
+        )
+
+        test = MFT(
+            **template,
+            labels=self._neutral,
+            name='"A is COMP than B" gives no information about "A is COMP than C"',
+            capability="NER",
+            description='Eg. "A is better than B" gives no information about "A is better than C"',
+        )
+
+        self.add_test(test)
+
+    def _default_temporal_tests(self, data: Optional[Iterable[Tuple]], num_test_cases=100):
+        template = self.editor.template(
+            (
+                "{first_name} works as {a:professions}",
+                "{first_name} used to work as a {professions}",
+            ),
+            nsamples=num_test_cases,
+            remove_duplicates=True,
+        )
+
+        template += self.editor.template(
+            (
+                "{first_name} {last_name} is  {a:professions}",
```

Comment:
Extra space, I'm guessing that might influence the results?

```suggestion
                "{first_name} {last_name} is {a:professions}",
```